src/api/endpoints/activity_types.py

```diff
@@ -35,11 +35,13 @@ def post(cls):
             validation_status_code = status_code or 400
             return response_builder(errors, validation_status_code)
 
-        activity_type = ActivityType(name=result["name"],
-                                     description=result["description"],
-                                     value=result["value"],
-                                     supports_multiple_participants=result["supports_multiple_participants"]
-                                     )
+        supprt_mutiple = result["supports_multiple_participants"]
```

Comment:
Typo at `supprt` meaning there is a typo copied over and over again.

Reply:
@Andretalik Will be fixed